pages/13.contributing/06.Writer/docs.md

```diff
@@ -32,6 +32,126 @@ Please note that we do check to ensure that articles are not copied from other l
 
 It helps if you can provide an image to use with the article. This must be royalty free or a license to use the image be granted to the Mautic Community. We use [Unsplash][unsplash] and [Pixabay][pixabay] to find royalty free images.
 
+#### Mautic Community Blog Writer & Editorial Guidelines
+
+**Note:**
+_This is a first draft of what we might consider using for writers guidelines to contribute to the mautic.org blog.  These are based on the work done by the [Joomla project](https://magazine.joomla.org/all/item/41 "Joomla project") ~ Ruth Cheesley_
+
+##### Content guidelines
+
+  - All posts must be in English, unless posted in a specific international area.  If posting in an international area, please use the language of that area, and not English.  
+  - Do not discuss illegal activities.  Our server is hosted in the USA and so it is subject to US law.  Please do not expose us to any unnecessary legal liability.
+  - Use your own words.  If you wish to use the words of somebody else, quote them, citing the source.  Plagiarism is unethical and is illegal in many countries.
+  - Do not use any copyrighted material unless it's released under a free use license such as Creative Commons or compatible.  If such copyrighted material is used, proper credit/attribution must be given.
+  - In case you use copyrighted material that demands authorization from the author, make sure you have that authorization before you post.
+  - Do not link to any site that contains adult content, sexually oriented material or might otherwise be considered offensive.  Any post containing an inappropriate link will be deleted and the poster will receive a warning.
```

Comment:
```suggestion
  - Do not link to any site that contains adult content, sexually oriented material or might otherwise be considered offensive. Any post containing an inappropriate link will be deleted and the poster will receive a warning.
```